src/vscode-cosmosdb.api.d.ts

```diff
@@ -51,6 +51,12 @@ export interface DatabaseAccountTreeItem extends CosmosDBTreeItem {
         masterKey: string;
         documentEndpoint: string;
     }
+
+    postgresData?: {
```

Comment:
Whenever you change the api, you also need to change the version:
https://github.com/microsoft/vscode-cosmosdb/blob/master/src/extension.ts#L175

Reply:
Would it be reasonable to make it 1.2.0 or 1.1.1?

Reply:
Per https://semver.org/

> 1. MAJOR version when you make incompatible API changes,
> 2. MINOR version when you add functionality in a backwards compatible manner, and
> 3. PATCH version when you make backwards compatible bug fixes.

This is definitely a functional change, not just a bug fix. So it's either 1.2.0 or 2.0.0, depending if we have breaking changes in the api

Reply:
Not sure if it is incompatible, doesn't really break older App Service versions. I would change it to 1.2.0 for now and update before release if needed.